src/EventStore.ClusterNode/Program.cs

```diff
@@ -46,7 +47,10 @@ await CreateHostBuilder(hostedService, args)
 				.ConfigureWebHostDefaults(builder =>
 					builder.UseKestrel(server => {
 							server.Listen(hostedService.Options.IntIp, hostedService.Options.IntHttpPort,
-								listenOptions => listenOptions.UseHttps(hostedService.Node.Certificate));
+								listenOptions => listenOptions.UseHttps(new HttpsConnectionAdapterOptions {
```

Comment:
Where is the client certificate validation done? How do we limit which CA's are valid?

Reply:
We currently accept all CA certificates installed on the system (the same applies to the replication channel). I'll try to add a `CACertificatesPath` configuration option which will be mandatory if we apply the "secure by default" principle.